pkg/oauthserver/server/crypto/random.go

```diff
@@ -0,0 +1,36 @@
+package crypto
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+)
+
+const bitsInByte = 8
+
+// RandomBits returns a random byte slice with at least the requested bits of entropy.
+// Callers should avoid using a value less than 256 unless they have a very good reason.
+func RandomBits(bits int) []byte {
+	size := bits / bitsInByte
+	if bits%bitsInByte != 0 {
+		size++
+	}
```

Comment:
what about:
```
size = int(math.Ceil(float64(bits)/8.0)
```
?
Also I think we all know there are 8 bits in a byte, the const sounds excessive, it's not like it can ever change ...

Reply:
I dropped the const, but the left the rest as-is.  I find the one liner unreadable.  Playing around with float casts and importing `math` is not what I want to see here.